CMakeLists.txt

```diff
@@ -754,9 +754,9 @@ add_library (seastar
 add_library (Seastar::seastar ALIAS seastar)
 
 if (Seastar_HAVE_EXECINFO)
-target_compile_definitions (seastar
-  PUBLIC Seastar_HAVE_EXECINFO)
-endif()
+  target_compile_definitions (seastar
+    PUBLIC Seastar_HAVE_EXECINFO)
+endif ()
```

Comment:
could you please squash this change into the one which introduced `Seastar_HAVE_EXECINFO`?

Reply:
okay, squashed